rules/resource_rules.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2018 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# rules:
+#   resource_types:
+#   - 'project'
+#   - 'bucket'
+#   - 'dataset'
+#   - 'instance'
+#   resource_trees:
+#   - type: 'project'
+#     resource_id: 'proj-1'
+#     children:
+#     - type: 'bucket'
+#       resource_id: 'bucket-1'
+#     - type: 'bucket'
+#       resource_type: 'bucket-2'
+#     - type: 'dataset'
+#       resource_id: 'proj-1:dataset1'
+#     - type: 'dataset'
+#       resource_id: 'proj-1:dataset2'
+#   - type: 'project'
+#     resource_id: 'proj-2'
+#     children:
+#     - type: 'bucket'
+#       resource_id: 'bucket-3'
+#     - type: 'dataset'
+#       resource_id: 'proj-2:dataset1'
+#     - type: 'instance'
+#       resource_id: '*'
```

Comment:
can we do regex matching like 'prod-*' here?

Reply:
Currently only supports '*'. Clarified in the doc.